generators/component/templates/component.ts

```diff
@@ -46,6 +88,11 @@ export class <%= component.class %> extends<% if (component.isExtendingMes) { %>
 
     //#endregion
 
+    /**
+     * Constructor
+     * 
+     * @param viewContainerRef the reference to the component view container
+     */
     constructor(viewContainerRef: ng.ViewContainerRef) { 
```

Comment:
Everytime I need to use a progress indicator, if I try to use the viewContainerRef.elementRef it doesn't work. So my vote is to use get by default the elemenRef and not the viewContainerRef.